TODO: Fix the relevant xmlunit-legacy to have these dependencies in POMs instead
--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -83,7 +83,7 @@
     jcip: "net.jcip:jcip-annotations:1.0",
     junit: 'junit:junit:4.12',
     junitPlatform: 'org.junit.platform:junit-platform-launcher:1.0.3',
-    xmlunit: 'xmlunit:xmlunit:1.3',
+    xmlunit: ['xmlunit:xmlunit-legacy:2.0', 'xmlunit:xmlunit:2.0'],
     equalsVerifier: 'nl.jqno.equalsverifier:equalsverifier:2.1.6',
     xbean: 'org.apache.xbean:xbean-reflect:3.4', //required by maven3 classes
     nativePlatform: 'net.rubygrapefruit:native-platform:0.14',
